frontend/src/components/Accounts/AccountDetailsAccountStatement.vue

```diff
@@ -7,7 +7,9 @@
 					<TableTh>Type</TableTh>
 					<TableTh v-if="breakpoint >= Breakpoint.XL">Reference</TableTh>
 					<TableTh align="right">Amount</TableTh>
-					<TableTh v-if="breakpoint >= Breakpoint.XXL" align="right">Balance</TableTh>
+					<TableTh v-if="breakpoint >= Breakpoint.XXL" align="right"
+						>Balance</TableTh
+					>
```

Comment:
What happened here? that formatting is a bit messed up, is it not? 👀

Reply:
I thought so too so fixed it, but the formatter reverted it :shrug: